allauth/account/app_settings.py

```diff
@@ -190,6 +190,15 @@ def LOGIN_ON_EMAIL_CONFIRMATION(self):
         return self._setting('LOGIN_ON_EMAIL_CONFIRMATION', True)
 
     @property
+    def LOGIN_CALLBACK_PROXY(self):
+        return self._setting('LOGIN_CALLBACK_PROXY', '')
+
+    @property
+    def LOGIN_PROXY_REDIRECT_WHITELIST(self):
+        return filter(len,
```

Comment:
Using list comprehension. Given that I'm altering the auth library itself, I'd rather err on the paranoid side of security, though I admittedly can't see any issue from having an inadvertant blank entry in the whitelist